test-programs/account-compression-test/tests/merkle_tree_tests.rs

```diff
@@ -192,26 +192,27 @@ async fn test_init_and_insert_into_nullifier_queue_default() {
 
 #[tokio::test]
 async fn test_init_and_insert_into_nullifier_queue_custom() {
-    for changelog_size in (1000..5000).step_by(1000) {
-        for queue_capacity in [5003, 6857, 7901] {
-            let roots_size = changelog_size * 2;
-            test_init_and_insert_into_nullifier_queue(
-                &StateMerkleTreeConfig {
-                    height: STATE_MERKLE_TREE_HEIGHT as u32,
-                    changelog_size,
-                    roots_size,
-                    canopy_depth: STATE_MERKLE_TREE_CANOPY_DEPTH,
-                    network_fee: Some(5000),
-                    rollover_threshold: Some(95),
-                    close_threshold: None,
-                },
-                &NullifierQueueConfig {
-                    capacity: queue_capacity,
-                    sequence_threshold: roots_size + SAFETY_MARGIN,
-                    network_fee: None,
-                },
-            )
-            .await;
+    for changelog_size in (1..=5000).step_by(1000) {
+        for roots_size in (changelog_size..=5000).step_by(1000) {
+            for queue_capacity in [5003, 6857, 7901] {
+                test_init_and_insert_into_nullifier_queue(
```

Comment:
I think this one makes sense to iterate over multiple capacities.
I don't know how the underlying test works but it's probably enough to just do a couple inserts and empty the queue no need to do a lot of transactions here.